### PR TITLE
Rebuild docs for speed benchmark

### PR DESCRIPTION
**Pull request by wangxingjun778**
1. Rebuild docs for speed benchmark
2. Update readme for Qwen2.5 speed benchmark
3. Fix some formatting issues with tables

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,16 +37,17 @@ In the past three months since Qwen2's release, numerous developers have built n
 
 ## News
 
-- 2024.09.19: We released the Qwen2.5 series. This time there are 3 extra model sizes: 3B, 14B, and 32B for more possibilities. Check our [blog](https://qwenlm.github.io/blog/qwen2.5) for more! 
-- 2024.06.06: We released the Qwen2 series. Check our [blog](https://qwenlm.github.io/blog/qwen2/)!
+- 🚀2024.10.31: We released the Qwen2.5 speed benchmark. Check our [doc](https://qwen.readthedocs.io/en/latest/benchmark/speed_benchmark.html) for more!
```
Comment (jklj077) [inline README.md:40]: currently only used for model releases

File: docs/locales/zh_CN/LC_MESSAGES/benchmark/speed_benchmark.po [hunk 2]
```diff
@@ -16,219 +16,250 @@ msgstr ""
...
+#, fuzzy
+msgid "Qwen2.5 Speed Benchmark"
 msgstr "效率评估"
 
 #: ../../source/benchmark/speed_benchmark.rst:5
-#: 3e97857c19314350b1d6686ad9776d35
-msgid "To be updated for Qwen2.5."
-msgstr "Qwen2.5结果待更新，由于模型结构差异有限，Qwen2结果可供参考。"
+#: 5577386104e04ce0820d75b8d4a4b9bb
+#, fuzzy
```
[the remark this answers is not in the document]
Comment (wangxingjun778) [reply docs/locales/zh_CN/LC_MESSAGES/benchmark/speed_benchmark.po]: done

File: docs/source/benchmark/speed_benchmark.rst [hunk 3]
```diff
@@ -11,21 +11,21 @@ The environment of the evaluation with huggingface transformers is:
...
 The environment of the evaluation with vLLM is:
 
 -  NVIDIA A100 80GB
 -  CUDA 12.1
--  vllm==0.6.3
--  torch==2.4.0
--  flash_attn==2.6.3
--  transformers==4.46.0
+-  vLLM 0.6.3
+-  Pytorch 2.4.0+cu121
```
[the remark this answers is not in the document]
Comment (wangxingjun778) [reply docs/source/benchmark/speed_benchmark.rst:26]: fixed

File: docs/source/benchmark/speed_benchmark.rst [hunk 4]
```diff
@@ -43,42 +43,41 @@ Notes:
...
-+                         +              +--------------+---------+-----------------+----------------+
-|                         |              | GPTQ-Int8    | 1       | 36.25           | 2.01           |
-+                         +              +--------------+---------+-----------------+----------------+
-|                         |              | GPTQ-Int4    | 1       | 49.22           | 1.85           |
-+                         +              +--------------+---------+-----------------+----------------+
-|                         |              | AWQ          | 1       | 36.90           | 1.84           |
-+-------------------------+--------------+--------------+---------+-----------------+----------------+
-
++-------------------------+--------------+--------------+---------+-----------------+----------------+---------------------------+
+| Model                   | Input Length | Quantization | GPU Num | Speed(tokens/s) | GPU Memory(GB) | Note                      |
```
[the remark this answers is not in the document]
Comment (wangxingjun778) [reply docs/source/benchmark/speed_benchmark.rst:47]: Got infer error for all qwen2.5 int8 models:
``` RuntimeError: probability tensor contains either `inf`, `nan` or element < 0  ```
under settings:
``` auto_gptq>=0.7.0  cu1210 ```

The env: ```auto_gptq==0.6.0+cu1210``` can run the infer code successfully.

**Pull request merged**